packages/dd-trace/src/appsec/reporter.js

```diff
@@ -116,8 +122,11 @@ function reportAttack (attackData) {
 
   newTags['appsec.event'] = 'true'
 
-  if (limiter.isAllowed()) {
-    newTags['manual.keep'] = 'true' // TODO: figure out how to keep appsec traces with sampling revamp
+  if (isStandaloneEnabled()) {
```

Comment:
we bypass the rate limiter when standalone billing is enabled ???

Reply:
yes, we want all events

Reply:
why is that ?

Reply:
ok, double checked and changed to its original behavior

Reply:
Aaaah see I told you it was weird!